particles/Processing/js/TopLabels.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ * Copyright (c) 2019 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+defineParticle(({DomParticle, resolver, log}) => {
+
+  importScripts(resolver(`$here/tf.js`));
+
+  return class extends self.TfMixin(DomParticle) {
+    async update({yHat, labels, k}) {
+      if (yHat && labels) {
+        const topK = k || 5;
+        log(`Converting tensor output to top ${topK} labels...`);
```

Comment:
I like the string interpolation here.

Reply:
That's good, because [you wrote it](https://github.com/PolymerLabs/arcs/blame/4b89c9eaa9041cac1ee9fb4bcaa64103fa2b1dba/particles/Processing/particles/js/TopLabels.js#L25). :)